community/contribute/elasticsearch-develop.md

```diff
@@ -4,22 +4,42 @@ title: "ElasticSearch notifier development"
 sidebar_position: 3
 ---
 
-The easiest way to develop Botkube with Elasticsearch notifier enabled is to install Elasticsearch on your local Kubernetes cluster.
+The easiest way to develop Botkube with Elasticsearch notifier enabled is to install Elasticsearch on your local Kubernetes cluster via [ECK](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-deploy-eck.html).
 
 ## Steps
 
 1. Install Elasticsearch:
 
-   ```bash
-   helm repo add elastic https://helm.elastic.co
-   helm install elasticsearch elastic/elasticsearch --version 7.17.3  --set replicas=1 --set resources.requests.cpu="100m" --set resources.requests.memory="512M" --wait
-   ```
+   1. Install ECK custom resource definitions:
+      ```bash
+      kubectl create -f https://download.elastic.co/downloads/eck/2.9.0/crds.yaml
+      ```
+   2. Install ECK operator:
+      ```bash
+      kubectl apply -f https://download.elastic.co/downloads/eck/2.9.0/operator.yaml
+      ```
+   3. Deploy Elasticsearch:
+      ```bash
+      cat <<EOF | kubectl apply -f -
+      apiVersion: elasticsearch.k8s.elastic.co/v1
+      kind: Elasticsearch
+      metadata:
+        name: elasticsearch
+      spec:
+        version: 8.9.1
+        nodeSets:
+        - name: default
+          count: 1
+          config:
+            node.store.allow_mmap: false
+      EOF
+      ```
 
 1. Install Botkube with Elasticsearch, according to the [Elasticsearch installation](../../installation/elasticsearch) instruction, where:
 
    - `ELASTICSEARCH_USERNAME` is `elastic`,
-   - `ELASTICSEARCH_PASSWORD` is `changeme`,
-   - `ELASTICSEARCH_ADDRESS` is `http://elasticsearch-master.default:9200`.
+   - `ELASTICSEARCH_PASSWORD` is `$PASSWORD`,
```

Comment:
I assume w need to retrieve the password first, right? I think it can be unclear for some, but the separate, dedicated v8 section with instruction will solve the issue 👍